__tests__/gpg.test.ts

```diff
@@ -58,12 +58,12 @@ describe('gpg', () => {
     });
   });
 
-  describe('getKeygrip', () => {
-    it('returns the keygrip', async () => {
+  describe('getKeygrips', () => {
+    it('returns the keygrips', async () => {
       await gpg.importKey(userInfo.pgp);
-      await gpg.getKeygrip(userInfo.fingerprint).then(keygrip => {
-        console.log(keygrip);
-        expect(keygrip).toEqual(userInfo.keygrip);
+      await gpg.getKeygrips(userInfo.fingerprint).then(keygrips => {
+        console.log(keygrips);
+        expect(keygrips[0]).toEqual(userInfo.keygrip);
```

Comment:
Add an assertion with the expected number of keygrips.

Also update `userInfo` def?

Reply:
hi @crazy-max 

what do you mean by updating userInfo def ?

Reply:
Hi @yanndegat

Update the [userInfo object](https://github.com/crazy-max/ghaction-import-gpg/blob/master/__tests__/gpg.test.ts#L21). `keygrip` is now an array not a string.

Reply:
updated. tell me if it's what you had in mind